draft-ietf-mls-protocol.md

```diff
@@ -5464,15 +5542,30 @@ Template:
 
 Initial contents:
 
-| Value            | Name                     | Message(s) | Recommended | Reference |
-|:-----------------|:-------------------------|:-----------|:------------|:----------|
-| 0x0000           | RESERVED                 | N/A        | N/A         | RFC XXXX  |
-| 0x0001           | application_id           | LN         | Y           | RFC XXXX  |
-| 0x0002           | ratchet_tree             | GI         | Y           | RFC XXXX  |
-| 0x0003           | required_capabilities    | GC         | Y           | RFC XXXX  |
-| 0x0004           | external_pub             | GI         | Y           | RFC XXXX  |
-| 0x0005           | external_senders         | GC         | Y           | RFC XXXX  |
-| 0xf000  - 0xffff | Reserved for Private Use | N/A        | N/A         | RFC XXXX  |
+| Value            | Name                     | Message(s) | R | Ref      |
+|:-----------------|:-------------------------|:-----------|:--|:---------|
+| 0x0000           | RESERVED                 | N/A        | - | RFC XXXX |
+| 0x0001           | application_id           | LN         | Y | RFC XXXX |
+| 0x0002           | ratchet_tree             | GI         | Y | RFC XXXX |
+| 0x0003           | required_capabilities    | GC         | Y | RFC XXXX |
+| 0x0004           | external_pub             | GI         | Y | RFC XXXX |
+| 0x0005           | external_senders         | GC         | Y | RFC XXXX |
+| 0x0A0A           | GREASE                   | KP, GI     | Y | RFC XXXX |
```

Comment:
Do we want some of the GREASE values to be only KP, some only GI, and some either KP or GI?

Reply:
That wouldn't change anything because correctly-implemented receivers will ignore them in both KP and GI